src/ILVerify/src/ILVerify.runtimeconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+        "configProperties": {
```

Comment:
This fine needs to look exactly like https://github.com/dotnet/corert/blob/master/src/ILCompiler/src/ilc.runtimeconfig.json

It is missing `runtimeOptions` scope. Once I have fixed that, debugging worked just fine for me.